website_event_contact/views/website_event_contact.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+       
+        <template id="website_event_contact_template" inherit_id="website_event.event_description_full">
+            <xpath expr="//div[@class='col-md-4 css_noprint']" position="replace">
```

Comment:
Wrong replacement.

You should just add the little part you need (the contacts one, right after Organizer). You are instead overwriting many things you don't need to. This means problems for sure.